src/rest_api/general_application.py

```diff
@@ -221,28 +167,21 @@ def post(self) -> object:
                 "From and to can not be equal"
             )
 
-        if not os.path.isfile(video_path):
+        if not (os.path.isfile(video_path) or os.path.isdir(video_path)):
             api.abort(
                 400,
                 "The given source path '{}' does not seem to exist"
                 .format(video_path)
             )
 
-        _thread.start_new_thread(
-            process_data_and_extract_emotions,
-            (
-                video_id,
-                video_path,
-                video_time_range
-            )
-        )
+        # Send request to the video API
 
         return {'Response': 'The request has been sent and should be'
                             ' updated in Splunk as soon as it is done.'}, 201
```

Comment:
I think we should update this to not include splunk.

Reply:
This can be removed when the actual endpoints are being finished :)